variables.tf

```
@@ -567,3 +567,9 @@ variable "role" {
   type        = string
   default     = null
 }
+
+variable "security_group_name" {
+  description = "Common name. Used as part of security_group names"
```

If a variable has a default, the description should start with `(Optional)`
```suggestion
  description = "(Optional) Common name. Used as security_group name prefix and `Name` tag"
```

---

please also apply to `/modules/security_groups/variables.tf`

---

i have updated code as per review.